src/types/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Enums } from '../enums'
+import { AccountID } from './account-id'
+import { Amount } from './amount'
+import { Blob } from './blob'
+const Field = Enums.Field
+const { Currency } = require('./currency')
```

Comment:
Why can't you use `import` syntax here?

Reply:
Keeping this PR minimal -- all of the files in this PR are actually just renames, not new files. This will be changed in a future PR